src/overlays/actors/ovl_En_Ganon_Mant/z_en_ganon_mant.c

```diff
@@ -94,7 +94,12 @@ static u16 sVerticesMap[GANON_MANT_NUM_STRANDS * GANON_MANT_NUM_JOINTS] = {
     MAP_STRAND_TO_VTX(3),  MAP_STRAND_TO_VTX(2),  MAP_STRAND_TO_VTX(1), MAP_STRAND_TO_VTX(0),
 };
 
-#include "ovl_en_ganon_mant_gfx.c"
+#define MANT_TEX_WIDTH 32
+#define MANT_TEX_HEIGHT 64
+
+static u64 sForceDmaAlignment = 0;
```

Comment:
I dont think this should mention anything "dma" related, this "boundary" is inside of a file as far as the dma table knows

Maybe just `sForceAlignment` idk

Reply:
I just moved over the already existing variable from the `_gfx.c` file. 
But you are right, I'll rename it